packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanel.tsx

```diff
@@ -438,7 +438,7 @@ export function TransferPanel() {
             l2Provider: l2Provider
           })
 
-          if (!allowance.gte(amountRaw)) {
+          if (!BigNumber.from(allowance).gte(amountRaw)) {
```

Comment:
Yeah, this was to fix a bug. 
Now that we have `await getL1TokenData` coming cached from session storage, the context of the `allowance` being a bigNumber (with all the bignum methods attached to it, etc.) is somehow lost in this case. 
From session storage, it only returns a plan `{hex: '....', isBignumber:true} `object. I needed to reconstruct this as a bignumber to make this code work, hence.
Otherwise, it resulted in an error - `allowance.gte is not a function...` and token deposits were failing.

Reply:
Make sense to me. @spsjvc you can resolve if it works for you

Reply:
Reverted this to the original implementation, as I have handled this now at the sessionStorage-get level. Otherwise would've needed to replicate this logic everywhere we try to fetch allowance in code.